### ISSUE TITLE
Some dependency breaks our tests

### ISSUE THREAD

**Issue by chrisbolin** [FormidableLabs/victory-native#123]
follow up on https://github.com/FormidableLabs/victory-native/issues/121

```
yarn install
yarn test # works

rm yarn.lock
yarn install # generates a new yarn.lock
yarn test # fails
```

**Comment by chrisbolin** [comment]
looks like it was `@jonny/react-native-mock` from `0.4.0` to `0.4.1`

**Comment by chrisbolin** [comment]
fixed in #124
